integ/link_test.go

```diff
@@ -66,6 +67,18 @@ func TestCreateLink(t *testing.T) {
 			ExpectStatus: http.StatusBadRequest,
 			ExpectBody:   `{"url": "This is not valid."}`,
 		},
+		{
+			Name:           "empty favicon should succeed",
+			GivenSessionID: existingUser.SessionID,
+			GivenBody: map[string]string{
+				"url":         "https://example.com",
+				"title":       "Testing",
+				"description": "It's only a test.",
+				"favicon":     "https://via.placeholder.com/16",
```

Comment:
This "favicon" field should point to an empty string, as the name of the tests says.